src/map/options.ts

```diff
@@ -72,6 +72,8 @@ export class MapOptions extends OptionsGroup {
         reverse: HTMLOption<'boolean'>;
     };
 
+    /// The HTML buttom to open the settings modal
+    private _openSettings: HTMLElement;
```

Comment:
What is the purpose of this file being included in this PR? These changes look like cleanups.

Reply:
This is a fix for one of the tests I added, which was failling, namely checking that calling `remove` doe indeed remove everything.

Reply:
Does it belong in this PR or should it be split off?

Reply:
I this it belongs to here, tests and corresponding fix are in the second commit